service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceLocators.scala

```diff
@@ -107,19 +108,18 @@ trait ConfigurationServiceLocatorComponents extends CircuitBreakerComponents {
 /**
  * A service locator that uses static configuration.
  */
-class ConfigurationServiceLocator(configuration: Configuration, circuitBreakers: CircuitBreakersPanel)(implicit ec: ExecutionContext)
+class ConfigurationServiceLocator(configuration: Config, circuitBreakers: CircuitBreakers)(implicit ec: ExecutionContext)
   extends CircuitBreakingServiceLocator(circuitBreakers) {
 
-  @deprecated(message = "Use constructor accepting {@link com.lightbend.lagom.scaladsl.client.CircuitBreakersPanel} instead", since = "1.4.0")
+  @deprecated(message = "prefer constructor using typesafe Config instead", since = "1.4.0")
```

Comment:
Looks like the merge conflict resulted in some changes reverted from what's on master.